corehq/apps/tour/tours.py

```diff
@@ -0,0 +1,33 @@
+from corehq.apps.tour.models import StaticGuidedTour, GuidedTourStep
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_noop as _
+
+
+SIMPLE_NEW_APP = StaticGuidedTour('simple_new_app', [
```

Comment:
There's a smell here. This is a global constant, but it has state (the cache). Among other things, this can make testing difficult.